framework/test-vectors/README.md

```diff
@@ -0,0 +1,113 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+## Summary
+
+These test vectors are intended to be used to validate interoperability
```

Comment:
I suspect I've already lost this battle but I still prefer the term "compatibility" to "interoperability".

Reply:
Can you say why?